### PR TITLE
v0.11.3-rc1

### PR DESCRIPTION
**Pull request by dehann**
EDIT: wait for these to be merged to master first
- [x] #726 
- [x] #728 -- error in tests, `todot`
- [x] #735

### CODE REVIEW

File: src/LightDFG/services/LightDFG.jl [hunk 1]
```diff
@@ -336,7 +336,7 @@ function savedot_attributes(io::IO, dfg::LightDFG)
         write(io, "$vl [color=red, shape=ellipse];\n")
     end
     for fl in listFactors(dfg)
-        write(io, "$fl [color=blue, shape=box];\n")
+        write(io, "$fl [color=blue, shape=box, fontsize=8, fixedsize=false, height=0.1, width=0.1];\n")
```
Comment (Affie) [inline src/LightDFG/services/LightDFG.jl:339]: Maybe the cause of this: #728

Comment (dehann) [reply src/LightDFG/services/LightDFG.jl:339]: That could make sense i did change this recently.  i wanted to shrink the factor boxes with GraphViz so added a few attributes.  Could we perhaps modify the test a little to allow that?

Comment (Affie) [reply src/LightDFG/services/LightDFG.jl:339]: My plan is to make time for coding on Friday, I'll update the tests if you haven't gotten to it by then.

Comment (dehann) [reply src/LightDFG/services/LightDFG.jl:339]: Linked issue and thanks for the fix!

Comment (dehann) [reply src/LightDFG/services/LightDFG.jl:339]: also see #735

**Pull request merged**